analytics/terraform/spark-eks-ipv6/examples/karpenter/nvme-yunikorn-gang-scheduling/taxi-trip-execute.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
You can remove this file and rerer to https://github.com/awslabs/data-on-eks/blob/main/analytics/scripts/taxi-trip-execute.sh